src/sacn/sacnsender.cpp

```diff
@@ -522,8 +522,13 @@ void CStreamServer::DEBUG_DESTROY_PRIORITY_UNIVERSE(uint handle)
 void CStreamServer::DestroyUniverse(uint handle)
 {
     QMutexLocker locker(&m_writeMutex);
-    if(handle < m_multiverse.size())
-        SetStreamTerminated(m_multiverse[handle].psend, true);
+    if (m_multiverse[handle].draft)
```

Comment:
Just glancing through and saw this -- this is a potentially unsafe vector access. The bounds check should be moved to be before any dereference.

Reply:
Good call